.github/workflows/publish.yml

```diff
@@ -19,9 +19,8 @@ jobs:
         with:
           dotnet-version: "6.0.x"
       
-      # TODO: Enable back once packages are ready to be published
-      # - run: ./Build.ps1
-      #   shell: pwsh
-      #   env:
-      #     NUGET_SOURCE: ${{ secrets.NUGET_SOURCE }}
-      #     NUGET_API_KEY: ${{ secrets.GSOFTINC_NUGET_API_KEY }}
+      - run: ./Build.ps1
+        shell: pwsh
+        env:
+          NUGET_SOURCE: ${{ secrets.NUGET_SOURCE }}
+          NUGET_API_KEY: ${{ secrets.GSOFTINC_NUGET_API_KEY }}
```

Comment:
There's something commented in the build.ps1 too! The place where we actually use the env vars